cfme/configure/tasks.py

```diff
@@ -39,7 +68,32 @@ def delete_all_tasks(destination):
     view.delete.item_select('Delete All', handle_alert=True)
 
 
-def is_task_finished(destination, task_name, expected_status, clear_tasks_after_success=True):
+def is_analysis_finished(name, task_type='vm', clear_tasks_after_success=True):
+    """ Check if analysis is finished - if not, reload page"""
+
+    tabs_data = TABS_DATA_PER_PROVIDER[task_type]
+    return all_tasks_analysis_pass(destination=tabs_data['tab'],
+                                   task_name=tabs_data['task'].format(name),
+                                   expected_status=tabs_data['state'],
+                                   expected_num_of_tasks=1,
+                                   silent_failure=False,
+                                   clear_tasks_after_success=clear_tasks_after_success)
+
+
+def are_all_tasks_match_status(name, expected_num_of_tasks, task_type='container'):
```

Comment:
**Required**

This is a method in a common module, and should not default to any task type.

Reply:
removed the default value for task_type